msal-dotnet-articles/TOC.yml

```diff
@@ -1,172 +1,181 @@
-- name: Microsoft Authentication Library for .NET
-  href: index.md
-- name: Getting started
+- name: Microsoft authentication libraries for .NET
```

Comment:
We should probably change this to "Choosing the right library"

Reply:
yes, let's do that!